scripts/common.py

```diff
@@ -45,7 +45,7 @@
 NODE_PATH = os.path.join(OPPIA_TOOLS_DIR, 'node-%s' % NODE_VERSION)
 NODE_MODULES_PATH = os.path.join(CURR_DIR, 'node_modules')
 FRONTEND_DIR = os.path.join(CURR_DIR, 'core', 'templates', 'dev', 'head')
-YARN_PATH = os.path.join(OPPIA_TOOLS_DIR, 'yarn-%s' % YARN_VERSION)
+YARN_PATH = os.path.join(OPPIA_TOOLS_DIR, 'yarn-%s' % YARN_VERSION[1:])
```

Comment:
Add a comment here that there's an assumption that the `YARN_VERSION` starts with a `v`.

Reply:
Done.

Reply:
Personally, I feel the value of `YARN_VERSION` should not contain the leading `v`. Also, other version variables do not have it.